docs/config.yaml

```diff
@@ -105,6 +105,10 @@ additional_arches: [aarch64, ppc64le, s390x]
 # OPTIONAL: environment variables to set inside the COSA container
 env:
   VAR: value
+  # This matches the UUID in multi-arch-builders/create-cex-device.sh
+  # Ideally, we'd pass this a cleaner way. See also:
+  # https://github.com/coreos/coreos-assembler/pull/3828#discussion_r1669010741
+  KOLA_CEX_UUID: "68cd2d83-3eef-4e45-b22c-534f90b16cb9"
```

Comment:
Remove. Move comment to `config.yaml`.

Reply:
Done

Reply:
Hi @jlebon ,

Removed from the docs and updated in the `config.yaml`.